node/node.go

```diff
@@ -22,7 +22,7 @@ import (
 	"github.com/ElrondNetwork/elrond-go/dataRetriever"
 	"github.com/ElrondNetwork/elrond-go/debug"
 	"github.com/ElrondNetwork/elrond-go/facade"
-	mainFactory "github.com/ElrondNetwork/elrond-go/factory"
+	factory2 "github.com/ElrondNetwork/elrond-go/factory"
```

Comment:
Please rename this or the other factory involved

Reply:
done